InnerEye/ML/configs/segmentation/ProstateBase.py

```diff
@@ -55,6 +55,7 @@ def __init__(self,
             ground_truth_ids_display_names=ground_truth_ids_display_names,
             colours=colours,
             fill_holes=fill_holes,
+            roi_intepreted_types=["ORGAN"] * len(fill_holes),
```

Comment:
can we use the same logic as for creating fill_holes? either use constructor arg or autopopulate?